[Feat] RO spider

dehatanes opened the pull request:
Implementando spider para o estado Rondonia.

Fontes usadas:
- Notícias em https://rondonia.ro.gov.br/?s=boletim+coronavirus (✅  total oficial ✅  valores por município ❌  valores de importados/indefinidos (estes não são disponibilizados na fonte))
- Painel com totais oficiais em https://covid19.sesau.ro.gov.br/ (✅  total oficial )

Fonte não usada (a ser implementada em PRs futuros):
- PDFs em https://rondonia.ro.gov.br/covid-19/noticias/relatorios-de-acoes-sci/

Commit:
[Feat] RO spider

turicas committed:
Implementa spider e parser de PDFs de Rondônia

O spider ainda está fora do formato padronizado

turicas commented:
Implementei a extração de PDFs (para PDFs desde 2021-01-18) no commit a82e5f8aa0192d91c4eb81143e856980f27b52df (`parsers/rondonia.py`) - além disso, fiz o parser dos links para os PDFs (`scripts/rondonia.py`). Também adicionei testes de extração para os PDFs que mais destoam dentro do padrão.
@dehatanes você poderia integrá-los ao spider e subir nesse PR, por favor? =)

Commit:
Move csv files from RO tests to the correct folder to do the assertion

Commit:
[Feat] RO pdf parser - scrap pdfs main page

Commit:
[Feat] Incorporate rondonia parser to rondonia spider

Commit:
Remove code that is already incorporated into spider

Commit:
Happy black :)

Commit:
Fix spider to parse old news format
Such as the table on https://rondonia.ro.gov.br/edicao-408-boletim-diario-sobre-coronavirus-em-rondonia/

dehatanes commented:
@turicas ✅ 
Eu removi nos commits código que teoricamente não seria necessário de acordo com os comentários que você deixou. Depois pode dar uma olhada para conferir se eu não interpretei nada errado e tirei coisas a mais, por favor?
